src/win/win_config_manager.h

```diff
@@ -3,13 +3,22 @@
 
 #include <string>
 #include "i_config_manager.h"
+#include "win_path_manager.h"
+#include <iostream>
+#include <fstream>
+
 
 class WinConfigManager : public IConfigManager {
     public:
-        WinConfigManager(){}
+        WinConfigManager();
         ~WinConfigManager(){}
         void writeConfig(const std::string key, const std::string value);
         std::string getConfig(const std::string key);
+
+    private:
+        void moveConfig();

```

Comment:
What is this function used for? There doesn't seem to be an implementation of it.

Reply:
This is right, there is no implementation. I forgot to remove this out of my commit